main_test.go

```diff
@@ -111,7 +111,7 @@ func Test_exportChangelog(t *testing.T) {
 		mockExporter.On("MaxEnvBytes").Return(0, nil).Once()
 		mockExporter.On("ExportEnv", mockContent).Return(errors.New("")).Once()
 
-		require.Error(t, exportChangelog(mockContent, mockExporter))
+		require.Error(t, exportChangelog(mockContent, mockExporter)) //nolint
```

Comment:
Yeah this is something I could not find an easy workaround for. 
Basically, there is a lint rule for passing things that are mutex (mockExporter) by value. Since this is test code, I opted to ignore rather than pick apart the tests.